boards/shields/xenvm_dom0/doc/index.rst

```diff
@@ -0,0 +1,53 @@
+.. _xenvm_dom0:
+
+Xen VM Dom0: universal shield for XEN VM Dom-0
+##############################################
+
+Overview
+********
+
+This virtual shield allows user to build Zephyr as a Xen initial domain (Dom0). The feature
+is implemented as shield to allow support for any compatible platform.
+
+How to add support of a new board
+*********************************
+
+* add correct memory node, based on region Xen picked for Domain-0 on your setup;
+* add mapping of hypervisor node to appropriate SoC;
```

Comment:
(note) looks like we don't need it anymore
@xakep-amatop FYI

Reply:
This node may be moved for different SoC, so this line is valid.

Reply:
yep, but MMU memory mapping is here meant under mapping, i.e. putting of the hypervisor node inside mmu_regions array, it isn't needed anymore, hypervisor node memory is mapped inside grant tables driver during initialization

but we need to add comment about adding hypervisor node for a board overlay inside this shield, as far as I see a common shield overlay has been deleted

Reply:
Oh, I got it. Reworded.